multicluster-hub-operator/uninstall.sh

```diff
@@ -41,6 +41,7 @@ oc get service | grep "multicluster" | awk '{ print $1 }' | xargs oc delete serv
 
 oc get scc | grep "multicluster" | awk '{ print $1 }' | xargs oc delete scc --wait=false --ignore-not-found
 oc get scc | grep "multicloud" | awk '{ print $1 }' | xargs oc delete scc --wait=false --ignore-not-found
+oc get scc | grep "kui-proxy" | awk '{ print $1 }' | xargs oc delete scc --wait=false --ignore-not-found
```

Comment:
See above.  I'll make another PR after to remove the other scc removaals in the unisntall.sh scripts.  

And thanks for your help @subbarao-meduri!  You reminded me and others about our hack-ish cleanup work in the 1.0 timeline 😄 and helped us clean it up!

Reply:
Fixed in the latest commit